features/admin.sms-providers.v1/pages/sms-providers.tsx

```diff
@@ -365,7 +363,6 @@ const SMSProviders: FunctionComponent<SMSProviderPageInterface> = (
             }).finally(() => {
                 setIsSubmitting(false);
                 mutateSMSProviderConfig();
-                mutateNotificationSendersFetchRequest();
```

Comment:
This was added to update the outdated values in other sms provider tabs, on updating an sms provider. For example, let's say a user already has configured twilio and later decides to switch to a custom provider. When this user updates the custom provider details and clicks on the twilio tab, the older twilio config values should no longer be displayed there. That requirement is satisfied by this mutation.

Hope that behaviour is not affected by this change, otherwise removing this would result in another bug report ;)

Please correct me if I'm missing something though.

Reply:
This is a duplicated with `mutateSMSProviderConfig`. There were 2 SWR hooks using the same endpoint commited previously. The duplicated one was removed.